docs/troubleshooting.md

```diff
@@ -0,0 +1,8 @@
+## Conflicting Package Names
+
+The Marketplace operator does not support reusing package names between two Operator Sources. To avoid unexpected behavior users should avoid adding any operator sources that reuse existing package names.
```

Comment:
`nit`: Isn't `OperatorSource`, a marketplace API, different than `Operator Sources`?

Reply:
In addition to space/no space and singular/plural, also make sure that capitalization is consistent.

Reply:
I'm not sure what you are trying to highlight in this nit. I don't see where it mentions `OperatorSource`.

Reply:
Are you saying that we should use `OperatorSource` instead of `Operator Sources`?

Reply:
@awgreene yes exactly that.

Reply:
Sure thing.